backend/src/dao/MembersDao.ts

```diff
@@ -125,6 +130,11 @@ export default class MembersDao extends BaseDao<IdolMember, IdolMember> {
     return Array.from(teamsMap.values());
   }
 
+  /**
+   * Gets a specific team by searching through each IDOL member and saving each member of the team.
+   * @param id The name of the team
+   * @returns A promise that resolves to the Team object with the specified name.
```

Comment:
This method could also return null if the team has no members. Can you specify that here as well?